README.md

```diff
@@ -1,2 +1,335 @@
-docket is a distributed background task system for Python functions with a focus
+Docket is a distributed background task system for Python functions with a focus
 on the scheduling of future work as seamlessly and efficiency as immediate work.
+
+[![PyPI - Version](https://img.shields.io/pypi/v/pydocket)](https://pypi.org/project/pydocket/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pydocket)](https://pypi.org/project/pydocket/)
+[![GitHub main checks](https://img.shields.io/github/check-runs/chrisguidry/docket/main)](https://github.com/chrisguidry/docket/actions/workflows/ci.yml)
+[![Codecov](https://img.shields.io/codecov/c/github/chrisguidry/docket)](https://app.codecov.io/gh/chrisguidry/docket)
+[![PyPI - License](https://img.shields.io/pypi/l/pydocket)](https://github.com/chrisguidry/docket/blob/main/LICENSE)
+
```

Comment:
Might be nice to bullet point features/what it does, what kind of tools it's maybe replacing.

![Screenshot 2025-03-05 at 4 05 25 PM](https://github.com/user-attachments/assets/09a28073-3ece-4791-aeb8-328ae92f9169)

![Screenshot 2025-03-05 at 4 10 49 PM](https://github.com/user-attachments/assets/52d81e8e-fbaf-464d-9cf0-9b679656f818)